src/Kunstmaan/NodeSearchBundle/Resources/config/services.yml

```diff
@@ -41,11 +41,14 @@ services:
         arguments: ['@service_container', '@kunstmaan_search.search', '%kunstmaan_node_search.indexname%', '%kunstmaan_node_search.indextype%', '%kunstmaan_node_search.search_configuration.number_of_shards%', '%kunstmaan_node_search.search_configuration.number_of_replicas%']
         calls:
             - [ setAclProvider, ['@security.acl.provider']]
-            - [ setIndexablePagePartsService, ['@kunstmaan_node_search.service.indexable_pageparts']]
         public: true
         tags:
             - { name: kunstmaan_search.search_configuration, alias: Node }
 
     kunstmaan_node_search.search.service:
         class: "%kunstmaan_node_search.search_service.class%"
         arguments: ['@service_container', '@request_stack']
+
+    kunstmaan_node_search.search_renderer:
```

Comment:
Should we use the FQCN and short service definition for new code parts? What do you think?

```yaml
Kunstmaan\NodeSearchBundle\Helper\SearchRenderHelper:
    arguments: ['@service_container', '@request_stack','@kunstmaan_node_search.service.indexable_pageparts']
```